pkgs/servers/nextcloud/default.nix

```diff
@@ -31,24 +31,19 @@ let
     };
   };
 in {
-  nextcloud17 = throw ''
-    Nextcloud v17 has been removed from `nixpkgs` as the support for it will be dropped
-    by upstream within the lifetime of NixOS 20.09[1]. Please upgrade to Nextcloud v18 by
+  nextcloud18 = throw ''
+    Nextcloud v18 has been removed from `nixpkgs` as the support for it was dropped
+    by upstream in 2021-01. Please upgrade to at least Nextcloud v19 by
     declaring
 
-        services.nextcloud.package = pkgs.nextcloud18;
+        services.nextcloud.package = pkgs.nextcloud19;
 
     in your NixOS config.
 
     [1] https://docs.nextcloud.com/server/18/admin_manual/release_schedule.html
   '';
 
-  nextcloud18 = generic {
-    version = "18.0.10";
-    sha256 = "0kv9mdn36shr98kh27969b8xs7pgczbyjklrfskxy9mph7bbzir6";
-    eol = true;
-  };
-
+  # FIXME(@Ma27) remove on 2021-06
```

Comment:
We might want to remove this earlier, as in before the 21.05 branch-off, because it will go EOL soon after.